### PR TITLE
feat: easter language toggle

### PR DESCRIPTION
**Pull request by NoamGaash**
# Description
type "english" to see the language toggle button.
it's temporary - once the english texts will be approved, we will release it without a secret code.

### CODE REVIEW

**Review by shootermv — APPROVED** [review]
good job!

File: src/layout/sidebar/menu/Menu.tsx [hunk 1]
```diff
@@ -22,19 +24,33 @@ function getItem(
   } as MenuItem
 }
 
-const MainMenu = () => {
+const LanguageToggle = () => {
```
Comment (shootermv) [inline src/layout/sidebar/menu/Menu.tsx:27]: can i ask you pls - to refactor this component to separate file?

**Review by shootermv — CHANGES_REQUESTED** [review]

File: src/layout/sidebar/menu/Menu.tsx (hunk 1 above)
Comment (NoamGaash) [reply src/layout/sidebar/menu/Menu.tsx:27]: Sure, you're right. I'll rearrange it later

**Comment by shootermv** [comment]
after thinking about more about the this change...
may be it can be good to split this `language toggle` change - to spaller subPoints

**Comment by NoamGaash** [comment]
sub components?

**Review by shootermv — APPROVED** [review]

**Pull request merged**